doc/api/net.md

```diff
@@ -226,6 +226,11 @@ added: v0.1.90
 Begin accepting connections on the specified `port` and `hostname`. If the
 `hostname` is omitted, the server will accept connections on any IPv6 address
 (`::`) when IPv6 is available, or any IPv4 address (`0.0.0.0`) otherwise.
+
+*Note*: Depending on the operating system, listening to an IPv6 address such
+as `'::'` may cause the `net.Server` to listen on an equivalent IPv4 address
```

Comment:
1. `to listen` -> `to also listen` (assuming that it is also not instead)
 2. What happens with normal IPv6 addresses (e.g. `2001:db8:85a3:0:0:8a2e:370:7334`)? Does an equivalent address get listened on as well, or is it just unspecified (`::`->`0.0.0.0`) and localhost (`::1`->`127.0.0.1`)?
 3. Could we be more specific about which OSs support this?